MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -97,25 +97,6 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::read(
         mesh = readBinary(file_name_base);
     }
 
-    auto* vtkGhostType =
-        mesh->getProperties().createNewPropertyVector<std::int8_t>(
-            "vtkGhostType", MeshLib::MeshItemType::Cell, 1);
-    if (!vtkGhostType)
-    {
-        WARN(
-            "Could not create PropertyVector for the cell values of "
-            "vtkGhostType in Mesh.");
-    }
-    else
-    {
-        vtkGhostType->reserve(_mesh_info.regular_elements +
-                              _mesh_info.ghost_elements);
-        std::fill_n(std::back_inserter(*vtkGhostType),
```

Comment:
`<algorithm>` included is not need, too.
The simplest way to revert a commit is to use `git revert`.

Reply:
Reverted by using git revert.